en/docs/get-started/configure-sms-otp-authenticator.md

```diff
@@ -0,0 +1,144 @@
+This document provides step by step instructions to set up SMS OTP configurations.
+
+## Configuring deployment.toml
+
+1. Open `<IS_HOME>/repository/conf/deployment.toml` file and configure the authenticator configurations as follows.
+
+     ``` toml
+     [open_banking.sca.idp]
+     name = "SMSAuthentication"
+     ```
+
+2. Open `<IS_HOME>/repository/conf/common.auth.script.js` file and update the configurations as follows.
+
+    ``` 
+    var psuChannel = 'Online Banking';
+
+    var onLoginRequest = function(context) {
+    publishAuthData(context, "AuthenticationAttempted", {'psuChannel': psuChannel});
+    executeStep(1, {
+    onSuccess: function (context) {
+    Log.info("Authentication Successful");
+    publishAuthData(context, "AuthenticationSuccessful", {'psuChannel': psuChannel});
+    OTPFlow(context);
+    },
+    onFail: function (context) {
+    Log.info("Authentication Failed");
+    publishAuthData(context, "AuthenticationFailed", {'psuChannel': psuChannel});
+    }
+    });
+    };
```

Comment:
Shall we format this code. We can use

Reply:
var psuChannel = 'Online Banking';

var onLoginRequest = function(context) {
    publishAuthData(context, "AuthenticationAttempted", {
        'psuChannel': psuChannel
    });
    executeStep(1, {
        onSuccess: function(context) {
            Log.info("Authentication Successful");
            publishAuthData(context, "AuthenticationSuccessful", {
                'psuChannel': psuChannel
            });
            OTPFlow(context);
        },
        onFail: function(context) {
            Log.info("Authentication Failed");
            publishAuthData(context, "AuthenticationFailed", {
                'psuChannel': psuChannel
            });
        }
    });
};

var OTPFlow = function(context) {
    executeStep(2, {
        //OTP-authentication
        onSuccess: function(context) {
            context.selectedAcr = "urn:openbanking:psd2:sca";
            publishAuthData(context, "AuthenticationSuccessful", {
                'psuChannel': psuChannel
            });
        },
        onFail: function(context) {
            publishAuthData(context, "AuthenticationFailed", {
                'psuChannel': psuChannel
            });
            OTPFlow(context);
        }
    });
};